project2/styles.css

```diff
@@ -0,0 +1,173 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Playfair Display', serif;
+    font-size: 17px;
+}
+
+
+.menu li {
+    display: inline-block;
+}
+
+ul {
+    text-align: left;
+    list-style: square;
+    padding-left: 60px;
+    padding-right: 40px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+.menu li+li {
+    margin-left: 110px;
+}
+
+li a {
+    background-color: rgb(255, 255, 255);
+    font-family: monospace;
+    color: orange;
+    font-size: 35px;
+    padding-left: 32px;
+
+}
+
+#fas fa-shopping-cart,
+#fas fa-utensil-spoon,
+#far fa-clock {
```

Comment:
Here, the selector is breaking because you can't have two `id`s (see notes on your HTML. You want to update this instead to:
```
#ingredients,
#instructions,
#time {
```